tagsets/helpers.php

```diff
@@ -136,6 +136,28 @@ function ortime__array_ampm_time_to_array_mil_time ($a) { // unused?
     return $r;
 }
 
+function ortime__get_weekday($unixtime,$language='') {
+    global $lang;
+    if (!$language) {
+        if (isset($lang['lang']) && $lang['lang']) {
+            $language=$lang['lang'];
+        } else {
+            global $expadmindata, $settings;
```

Comment:
I think global can go to the first line of the function:
`global $lang, $expadmindata, $settings;`